modules/samples/authenticators/README.md

```diff
@@ -0,0 +1,46 @@
+README
+================================
+
+Intro
+---------
+This describes the steps to configure and run the sample authenticators with Identity Server. 
+
+Prerequisites
+--------------
+
+Build Identity Server 5.5.0
+
+Tomcat 7 or above - To deploy the sample authenticator web app
+
+
+Configuration
+--------------
+
+1. Copy org.wso2.carbon.identity.sample.extension.authenticators-5.5.0-SNAPSHOT.jar to 
```

Comment:
Is there a way to parameterize this ?. Or else we can just remove the version and say it in an abstract way. Unless it's confusing to see a particular version which is not getting updated with respective sample version